core/domain/takeout_service_test.py

```diff
@@ -567,38 +569,40 @@ def test_export_data_nontrivial(self):
         thread_and_message_ids = self.GENERAL_FEEDBACK_THREAD_IDS
         thread_and_message_ids.append(thread_id)
 
-        activities = self.ACTIVITY_IDS
-        activities.extend(self.EXPLORATION_IDS)
-
-        subscriptions_data = {
+        expected_subscriptions_data = {
             'creator_ids': self.CREATOR_IDS,
             'collection_ids': self.COLLECTION_IDS,
             'activity_ids': self.ACTIVITY_IDS,
             'general_feedback_thread_ids': thread_and_message_ids,
             'last_checked': None
         }
 
+        activities = self.ACTIVITY_IDS
```

Comment:
What's this for? This and the next line seem unused.

Reply:
I can see how this is a bit confusing, so I moved things around a bit. In Python all assignment is by reference, so here I am extending the ACTIVITY_IDS to include the EXPLORATION_IDS. I rearranged the code so that this is more clear.